cli/cosipbftcontroller/mod.go

```diff
@@ -36,7 +36,11 @@ import (
 	"golang.org/x/xerrors"
 )
 
-const privateKeyFile = "private.key"
+const (
+	privateKeyFile = "private.key"
+	 errInjector = "injector: %v"
+	)
```

Comment:
seems like your IDE is not formatting your go code for you. Please make sure that your IDE is properly configured for Go, it should format your code with `gofmt` (the standard go code formatter) automatically. 
You can find some instructions here: https://github.com/dedis/popstellar/tree/master/be1-go#configure-your-environment.